src/main/scala/org/broadinstitute/dsde/firecloud/EntityService.scala

```diff
@@ -373,6 +373,14 @@ class EntityService(rawlsDAO: RawlsDAO, importServiceDAO: ImportServiceDAO, goog
     importServiceDAO.importJob(workspaceNamespace, workspaceName, importRequest, isUpsert = true)(userInfo)
   }
 
+  def listJobs(workspaceNamespace: String, workspaceName: String, runningOnly: Boolean, userInfo: UserInfo): Future[List[ImportServiceListResponse]] = {
+    // get jobs from Import Service
+    val importServiceJobs = importServiceDAO.listJobs(workspaceNamespace, workspaceName, runningOnly)(userInfo)
+    // TODO AJ-1602: get jobs from cWDS
+    // TODO AJ-1602: merge lists before replying
+    importServiceJobs
+  }
+
```

Comment:
This is where, in future PRs, we'll do all the work of aggregating cWDS and Import Service job info